packages/simple-form/src/main.js

```diff
@@ -91,6 +91,7 @@ SimpleFormApp.propTypes = {
   cancelText: PropTypes.string,
   form: PropTypes.object.isRequired,
   model: PropTypes.object.isRequired,
+  stopValidation: PropTypes.bool,
```

Comment:
new app params should be documented in the app readme

shouldn't we name it just  `validation` (true/false)?